web/react/components/post_info.jsx

```diff
@@ -30,9 +30,7 @@ module.exports = React.createClass({
                         <div className="dropdown">
                         { isOwner || (this.props.allowReply === "true" && type != "Comment") ?
                             <div>
-                                <a href="#" className="dropdown-toggle theme" type="button" data-toggle="dropdown" aria-expanded="false">
-                                    [...]
-                                </a>
+                                <a href="#" className="dropdown-toggle theme" type="button" data-toggle="dropdown" aria-expanded="false" />
```

Comment:
Just confirmed and self-closing anchor tags aren't valid for HTML5 even though Chrome is smart enough to render it right. It's just treated as an opening tag. This should just be an empty anchor since I assume we're trying to follow specs if possible.